e2e/tests/accessibility.spec.ts

```diff
@@ -5,8 +5,7 @@ import { expect, test } from '../fixtures'
 // https://github.com/alphagov/govuk-frontend/issues/1280#issuecomment-509588851
 const DISABLED_RULES: Record<string, Record<string, string[]>> = {
   accordion: {
-    default: ['aria-allowed-attr'],
-    'with-summary-lines': ['aria-allowed-attr'],
+    '*': ['aria-allowed-attr'],
```

Comment:
It was getting a bit annoying having it fail every time I added an example – so added support for `*` meaning all examples for the component.